src/main/java/de/chojo/repbot/config/elements/PresenceSettings.java

```diff
@@ -0,0 +1,69 @@
+package de.chojo.repbot.config.elements;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.chojo.jdautil.localization.util.Replacement;
+import net.dv8tion.jda.api.entities.Activity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+@SuppressWarnings("FieldMayBeFinal")
```

Comment:
```suggestion
```

Reply:
This is required. Otherwise intellij will convert the fields to final fields, which will destroy the jacksons deserialization.